### DISCUSSION TITLE
Change of default text on default submit box

### DISCUSSION THREAD

**Discussion by JanSandtro** [hizzle-co/noptin#908] [Q&A]
I have trouble with the standard submit forms, so I only use a standard box without any formatting with shortcode [noptin]. Does anybody know how to change the name on the subscription button to something else?

**Comment by picocodes** [comment]
Hi @JanSandtro,

Try:-

```
[noptin submit="Subscribe Now" email_label="Email Address"]
```

Further reading:- https://noptin.com/guide/subscription-forms/newsletter-subscription-shortcode/#other-attributes
